controllers/categoriesController.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable prettier/prettier */
+const db = require("../models/category");
```

Comment:
Does not work as written.
```suggestion
const db = require("../models");
```
above code fixes file